internal/servers/worker/handler.go

```diff
@@ -165,114 +180,18 @@ func (w *Worker) handleProxy() http.HandlerFunc {
 		}
 		if err := wspb.Write(connCtx, conn, handshakeResult); err != nil {
 			event.WriteError(ctx, op, err, event.WithInfoMsg("error sending handshake result to client"))
-			conn.Close(websocket.StatusProtocolError, "unable to send handshake result")
+			_ = conn.Close(websocket.StatusProtocolError, "unable to send handshake result")
 			return
 		}
 
-		switch conn.Subprotocol() {
-		case globals.TcpProxyV1:
-			w.handleTcpProxyV1(connCtx, clientAddr, conn, si, ci.id, endpoint)
-		default:
-			conn.Close(websocket.StatusProtocolError, "unsupported-protocol")
-			return
-		}
-	})
+		handleProxyFn(connCtx, clientAddr, conn, nil, sessClient, si, ci.Id, endpoint)
+	}
 }
 
-func (w *Worker) wrapGenericHandler(h http.Handler, props HandlerProperties) http.Handler {
+func (w *Worker) wrapGenericHandler(h http.Handler, _ HandlerProperties) http.Handler {
 	return http.HandlerFunc(func(wr http.ResponseWriter, r *http.Request) {
 		// Set the Cache-Control header for all responses returned
 		wr.Header().Set("Cache-Control", "no-store")
 		h.ServeHTTP(wr, r)
 	})
 }
-
-/*
-func WrapForwardedForHandler(h http.Handler, authorizedAddrs []*sockaddr.SockAddrMarshaler, rejectNotPresent, rejectNonAuthz bool, hopSkips int) http.Handler {
```

Comment:
Hah, you finally took this out. We may need to add it back at some point, but we can deal with it then.

Reply:
Yeah I was not sure if this was needed down the road, but I am not a fan of commented out code.  Happy to put back if we will need it in the future